source/usage-examples/command.txt

```diff
@@ -23,6 +23,9 @@ The ``command()`` method returns a :mdn:`Promise
 <Web/JavaScript/Reference/Global_Objects/Promise>`. The Promise returns
 the results of the command operation when it completes.
 
+.. include:: /includes/connect-guide-note.rst
+
+
```

Comment:
While we're add it, why don't we add an "Example" heading to each code snippet, since there are some code snippets that have that already (some with exposition). I'd suggest the following formula:
```
Example
------------

my text talking about the example doing exampley things

<note>

<code snippet>
```

Point being: if there's an "example" heading, I think it's a little weird for this note to show up *before* the heading, since that's not usually how we use notes in other places in the docs.

Reply:
Not against this, but I'd prefer to add this to a separate PR

Reply:
Fixed